src/components/entry/Lightbox.js

```diff
@@ -0,0 +1,150 @@
+import { Dialog } from '@reach/dialog'
+import { LeftArrowAlt, RightArrowAlt, X } from '@styled-icons/boxicons-regular'
+import styled from 'styled-components/macro'
+
+import ImagePreview from '../ui/ImagePreview'
+import ResetButton from '../ui/ResetButton'
+import Review, { StyledImagePreview } from './Review'
+
+const StyledDialog = styled(Dialog)`
+  display: flex;
+  width: 80%;
+  max-width: fit-content;
```

Comment:
The implication of this max-width is that the children determine the size of the parent. This seems to work, but lets think more about what that suggests about how the Dialog should work (should it fit content or fit the page as a whole?)